src/Microsoft.DotNet.ProjectJsonMigration/project.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+    "warningsAsErrors": true
+  },
+  "dependencies": {
+    "Microsoft.DotNet.Compiler.Common": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.Cli.Utils": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.ProjectModel": {
+      "target": "project"
+    },
+    "Microsoft.Win32.Registry": {
+      "version": "4.0.0",
+      "exclude": "compile"
+    },
+    "MSBuild": "0.1.0-preview-00029-160805",
```

Comment:
In my struggle to workaround the `MSBUILD_EXE_PATH` error in my unit tests I added all these references. 

That is fixed with a text file named msbuild.exe now, so I will remove these.